cernopendata/modules/fixtures/data/records/jade-computing-notes.json

```diff
@@ -0,0 +1,5267 @@
+[
+  {
+    "abstract": {
+      "description": "Index"
+    },
+    "authors": "",
+    "accelerator": "DESY-PETRA",
+    "collaboration": {
+      "name": "JADE collaboration",
+      "recid": "451"
+    },
+    "collections": [
+      "JADE-Computing-Notes"
+    ],
+    "date_created": [
+      ""
+    ],
+    "date_published": "2023",
+    "distribution": {
+      "formats": [
+        "pdf"
+      ],
+      "number_files": 1,
+      "size": 137529
+    },
+    "experiment": "JADE",
+    "files": [
+      {
+        "checksum": "adler32:46c1eb5f",
+        "size": 137529,
+        "uri": "root://eospublic.cern.ch///eos/opendata/jade/documentation/notes/0Index.pdf"
+      }
+    ],
+    "license": {
+      "attribution": "CC0"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "261300",
+    "title": "JADE Computing Note 0",
+    "type": {
+      "primary": "Supplementaries",
+      "secondary": [
+        "Computing Note"
+      ]
+    }
+  },
+  {
+    "abstract": {
+      "description": "Monte Carlo Formats"
+    },
+    "authors": "",
```

Comment:
Empty "authors" field does not have to be there. You can remove it, the field is not mandatory.